ruma-client-api/src/r0/backup/add_backup_keys.rs

```diff
@@ -0,0 +1,44 @@
+//! [PUT /_matrix/client/r0/room_keys/keys](https://matrix.org/docs/spec/client_server/unstable#put-matrix-client-r0-room-keys-keys)
+
+use std::collections::BTreeMap;
+
+use js_int::UInt;
+use ruma_api::ruma_api;
+use ruma_identifiers::RoomId;
+use serde::{Deserialize, Serialize};
+
+ruma_api! {
+    metadata: {
+        description: "Store several keys in the backup.",
+        method: PUT,
+        name: "add_backup_keys",
+        path: "/_matrix/client/r0/room_keys/keys",
+        rate_limited: true,
+        requires_authentication: true,
+    }
+
+    request: {
+        /// The backup version. Must be the current backup.
+        #[ruma_api(query)]
+        pub version: String,
+        /// A map from room IDs to session IDs to key data.
+        pub rooms: BTreeMap<RoomId, Sessions>, // TODO: synapse has the sessions:{} wrapper, the spec has not
+    }
+
+    response: {
+        /// An opaque string representing stored keys in the backup. Clients can compare it with
+        /// the etag value they received in the request of their last key storage request.
+        pub etag: String,
+        /// The number of keys stored in the backup.
+        pub count: UInt,
+    }
+
+    error: crate::Error
+}
+
+/// TODO: remove
```

Comment:
@timokoesters Why are new types with doc (!) comments that say `TODO: remove` being added?

Reply:
See the comment a few lines above: `// TODO: synapse has the sessions:{} wrapper, the spec has not`

Reply:
Okay. @DevinR528 can you turn the above into a doc comment and put that here? `TODO: remove` should probably stay, but as a regular comment.